gunicorn/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # CHANGELOG - gunicorn
 
-1.0.003-22-2017
-==================
+## 1.1.005-23-2017
```

Comment:
Agreed, better fixing all at once. Also it would be nice to fix markdown syntax, now we have a mix of rst and md.